torch/csrc/jit/codegen/cuda/lower_unroll.cpp

```diff
@@ -264,9 +264,7 @@ bool UnrollPass::canOmitElseClause(kir::ForLoop* fl) {
   return true;
 }
 
-// Generate the loop nest structure and place it in lowered_exprs
 UnrollPass::UnrollPass(const std::vector<Expr*>& exprs) {
-  FUSER_PERF_SCOPE("GpuLower::Lower::UnrollPass::computeMap");
```

Comment:
We already have a perf scope for `runPass` and `runPass` just calls this function, so I don't think we need a separate perf scope here.